docs/tests.md

```diff
@@ -0,0 +1,18 @@
+# Tests for actors
+
+You can implement two types of tests for an actor in the leapp-repository - functional and unit tests.
+
+## Functional tests
+   - These tests provide fabricated input messages for the actor, check the output messages and check for side effects stated in the actor's description.
+   - These tests should consider the actor to be a black box, so ideally they should written by somebody who doesn't know the code.
+   - Example of a functional test: https://github.com/oamg/leapp-repository/blob/master/repos/system_upgrade/el7toel8/actors/redhatsignedrpmcheck/tests/test_redhatsignedrpmcheck.py
+
+## Unit tests
+   - These tests deal with individual actor's functions/methods.
+   - You can't unit test any method/function within the actor.py. You can write unit tests only for the functions/methods within actor's libraries.
+   - Thus, to be able to write unit tests for an actor, ideally the only thing in the actor.py's _process()_ method is calling the entry-point function of the actor's library python module.
+   - Tutorial on how to write unit tests: https://leapp.readthedocs.io/en/latest/unit-testing.html
+
+
+Both types of tests need to reside in the actor's _tests_ folder.
```

Comment:
Require only component tests, not unit tests. It may happen that an actor includes logic only in the actor's _process()_ method and then there would be little or no distinction between component and unit tests.

Reply:
What is required and what not should be mentioned in the contribution guidelines. This docs it about describing the two test levels.